pyoes/templates/pyoes/macros.jinja2

```diff
@@ -1,16 +1,17 @@
 {% macro plausible_tracker(plausible_data_domain, plausible_omgeving, extended=False) %}
-    {% if plausible_omgeving in ('dev', 'test') %}
-      {% set omgeving = plausible_omgeving + '-' %}
-    {% else %}
-      {% set omgeving = '' %}
-    {% endif %}
-    <!-- Plausible Analytics -->
-    {% if not extended %}
-    <script async defer data-domain="{{ plausible_data_domain }}" src="https://{{ omgeving }}webstats.onroerenderfgoed.be/js/script.js"></script>
-    {% else %}
-    <script async defer data-domain="{{ plausible_data_domain }}" src="https://{{ omgeving }}webstats.onroerenderfgoed.be/js/script.file-downloads.js"></script>
-    {% endif %}
-    <!-- End Plausible Analytics -->
+  {% if plausible_omgeving in ('dev', 'test') %}
+    {% set omgeving = plausible_omgeving + '-' %}
+    {% set src = "https://{{ omgeving }}webstats.onroerenderfgoed.be/js/script" %}
```

Comment:
detail: was ff in de war omdat die "script" hier mee in de src staat, code zou iets leesbaarder zijn denk ik als die gewoon verder als script.js staat

Reply:
Ja maar dan kunt ge die download tracking niet zo simpel toevoegen zoals nu

Reply:
Maar oké, ik heb het aangepast

Reply:
https://plausible.io/js/script deze ook nog :sweat_smile:

Reply:
Uw comment stond daar over in VS Code (ik steek alles op VS Code vanaf nu)